pypesto/optimize/ess/sacess.py

```diff
@@ -178,11 +230,12 @@ def minimize(
 
         Returns
         -------
-        Result object with optimized parameters in
-        :attr:`pypesto.Result.optimize_result`.
-        Results are sorted by objective. At least the best parameters are
-        included. Additional results may be included - this is subject to
-        change.
+        _:
```

Comment:
Why? Looks weird
![image](https://github.com/user-attachments/assets/9b31295b-42d1-48f1-a71e-780a7a3c3e79)

Reply:
Yes. Looks worse without it. That's because we use the napoleon/numpy-style docstrings that expect some name for the return values. I don't really like making up names there. (numpy uses [`out:`](https://numpy.org/doc/stable/reference/generated/numpy.array.html)).

Reply:
Makes sense, I don't get the point of a return value name. Then, fine as is :+1:

Reply:
Initially, it might have been the return type which was then replaced by something arbitrary to avoid redundancy with the Python return value annotations. Not completely sure.